packages/commonwealth/client/scripts/views/components/poll_card.tsx

```diff
@@ -0,0 +1,289 @@
+/* @jsx m */
+
+import m from 'mithril';
+
+import 'components/poll_card.scss';
+
+import { CWCard } from './component_kit/cw_card';
+import { CWButton } from './component_kit/cw_button';
+import { CWProgressBar } from './component_kit/cw_progress_bar';
+import { CWIcon } from './component_kit/cw_icons/cw_icon';
+import { CWCheckbox } from './component_kit/cw_checkbox';
+import { CWRadioButton } from './component_kit/cw_radio_button';
+import { CWText } from './component_kit/cw_text';
+
+export type VoteInformation = {
+  label: string;
+  value: string;
+  voteCount: number;
+};
+
+export function buildVoteDirectionString(voteOption: string) {
+  return `You voted "${voteOption}"`;
+}
+
+export type PollOptionAttrs = {
+  multiSelect: boolean;
+  voteInformation: Array<VoteInformation>;
+  selectedOptions: Array<string>;
+};
+
+export class PollOptions implements m.ClassComponent<PollOptionAttrs> {
+  view(vnode) {
+    const { multiSelect, voteInformation, selectedOptions } = vnode.attrs;
+    return (
+      <div class="PollOptions">
+        {multiSelect
+          ? voteInformation.map((option) => (
+              <CWCheckbox
+                checked={false}
+                label={option.label}
+                onchange={() => {
+                  // TODO: Build this out when multiple vote options are introduced.
+                  // Something like: this.selectedOptions.push(option.value);
+                  console.log('A vote for multiple options');
+                }}
+              />
+            ))
+          : voteInformation.map((option) => (
+              <CWRadioButton
+                checked={
+                  selectedOptions.length > 0 &&
+                  option.value === selectedOptions[0]
+                }
+                groupName="votes"
+                onchange={() => {
+                  selectedOptions[0] = option.value;
+                }}
+                label={option.label}
+                value={option.value}
+              />
+            ))}
+      </div>
+    );
+  }
+}
+
+export type CastVoteAttrs = {
+  disableVoteButton: boolean;
+  timeRemainingString: string;
```

Comment:
can just be `timeRemaining: string;`. no need to be redundant here bc it's typed